crates/worker-proto/proto/nemo/relay/worker/v1/plugin_worker.proto

```
@@ -45,6 +45,7 @@ enum RegistrationSurface {
   REGISTRATION_SURFACE_UNSPECIFIED = 0;
   SUBSCRIBER = 1;
   EVENT_METADATA_INJECTOR = 2;
+  EXPORT_ACTIVATION_POLICY = 3;
```

Buf (1.72.0)

[error] Enum value name "EXPORT_ACTIVATION_POLICY" should be prefixed with "REGISTRATION_SURFACE_".

(ENUM_VALUE_PREFIX)